consensus/state_sync/module.go

```diff
@@ -34,9 +25,14 @@ type StateSyncModule interface {
 	HandleGetBlockResponse(*typesCons.GetBlockResponse) error
 
 	IsServerModEnabled() bool
-	EnableServerMode()
+	EnableServerMode() error
+	DisableServerMode() error
 
 	SendStateSyncMessage(msg *typesCons.StateSyncMessage, nodeAddress cryptoPocket.Address, height uint64) error
+	IsOutOfSync() bool
+	AggregateMetadataResponses() error
+	GetAggregatedSyncMetadata() *typesCons.StateSyncMetadataResponse
+	Sync() error
```

Comment:
I think this should be `StartSynching` that triggers it to start requesting blocks from others